From vapier@gentoo.org  Tue Dec 22 12:03:40 2009
From: Michael Hennerich <michael.hennerich@analog.com>
Date: Mon, 21 Dec 2009 11:41:08 -0500
Subject: Driver core: export platform_device_register_data as a GPL symbol
To: Greg Kroah-Hartman <gregkh@suse.de>
Cc: Andrew Morton <akpm@linux-foundation.org>, linux-kernel@vger.kernel.org, Michael Hennerich <michael.hennerich@analog.com>
Message-ID: <1261413668-30145-1-git-send-email-vapier@gentoo.org>


From: Michael Hennerich <michael.hennerich@analog.com>

This allows MFD's to register/bind drivers for their sub devices while
still being compiled as a module.

Signed-off-by: Michael Hennerich <michael.hennerich@analog.com>
Signed-off-by: Mike Frysinger <vapier@gentoo.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/platform.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/base/platform.c
+++ b/drivers/base/platform.c
@@ -441,6 +441,7 @@ error:
 	platform_device_put(pdev);
 	return ERR_PTR(retval);
 }
+EXPORT_SYMBOL_GPL(platform_device_register_data);
 
 static int platform_drv_probe(struct device *_dev)
 {
